Tools/YamlCreate.ps1

```diff
@@ -2077,6 +2077,26 @@ Function Get-DebugString {
   return $debug
 }
 
+Function Write-ManifestContent {
+  Param
+  (
+    [Parameter(Mandatory = $true, Position = 0)]
+    [string] $FilePath,
+    [Parameter(Mandatory = $true, Position = 1)]
+    [PSCustomObject] $YamlContent,
+    [Parameter(Mandatory = $true, Position = 2)]
+    [string] $Schema
+  )
+  [System.IO.File]::WriteAllLines($FilePath, @(
+      $ScriptHeader + $(Get-DebugString);
+      "# yaml-language-server: `$schema=$Schema)";
+      '';
+      $(ConvertTo-Yaml $YamlContent).TrimEnd() -replace "(.*)$([char]0x2370)", "# `$1"
```

Comment:
I would suggest adding a comment explaining how the regex is working (commenting out special characters?)